src/Dflydev/FigCookies/Cookie.php

```diff
@@ -42,46 +42,42 @@ public function withValue(?string $value = null) : Cookie
 
     /**
      * Render Cookie as a string.
-     *
      */
-    public function __toString() : string
+    public function __toString(): string
     {
         return urlencode($this->name) . '=' . urlencode((string) $this->value);
     }
 
     /**
      * Create a Cookie.
-     *
      */
-    public static function create(string $name, ?string $value = null) : Cookie
+    public static function create(string $name, ?string $value = null): Cookie
     {
-        return new static($name, $value);
+        return new self($name, $value);
     }
 
     /**
      * Create a list of Cookies from a Cookie header value string.
      *
      * @return Cookie[]
      */
-    public static function listFromCookieString(string $string) : array
+    public static function listFromCookieString(string $string): array
     {
         $cookies = StringUtil::splitOnAttributeDelimiter($string);
 
-        return array_map(function ($cookiePair) {
+        return array_map(static function ($cookiePair) {
             return static::oneFromCookiePair($cookiePair);
         }, $cookies);
     }
 
     /**
      * Create one Cookie from a cookie key/value header value string.
-     *
      */
-    public static function oneFromCookiePair(string $string) : Cookie
+    public static function oneFromCookiePair(string $string): Cookie
     {
-        list ($cookieName, $cookieValue) = StringUtil::splitCookiePair($string);
+        [$cookieName, $cookieValue] = StringUtil::splitCookiePair($string);
 
-        /** @var Cookie $cookie */
-        $cookie = new static($cookieName);
+        $cookie = new self($cookieName);
```

Comment:
class is not final, means this is a behavior change

Reply:
Reverted.

Reply:
this one isn't reverted yet

Reply:
Indeed, can this one get updated, too?